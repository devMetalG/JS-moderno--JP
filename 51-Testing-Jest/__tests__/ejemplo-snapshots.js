const cliente = {
  nombre: 'Maguito',
  balance:800,
  tipo: 'Premium'
}
describe('test a cliente con snapshots', ()=>{
  test('Es memo',()=>{
    expect(cliente).toMatchSnapshot()
  })
})