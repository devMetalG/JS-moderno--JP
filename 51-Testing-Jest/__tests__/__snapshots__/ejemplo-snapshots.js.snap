// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test a cliente con snapshots Es memo 1`] = `
{
  "balance": 800,
  "nombre": "Maguito",
  "tipo": "Premium",
}
`;
